config/main

```diff
@@ -0,0 +1 @@
+include config/default
```

Comment:
Contrary to UI and API, we will not have one makefile per developer.
Can't we simply use docker compose files?